script.js

```diff
@@ -0,0 +1,32 @@
+function initHistogram() {
+	const dataInput = document.getElementById('dataInput').value;
+	const inputDataArray = dataInput.split(' ').filter(Number);
+
+	if (inputDataArray.length === 0) {
+		alert("Введите данные!")
+		return;
+	}
+
+	drawHistogram(inputDataArray);
+}
+
+function drawHistogram(dataArray) {
+	const histogram = document.createElement('div');
+	const maxWidth = (window.screen.width / 100) * dataArray.length;
+	const maxElement = Math.max(...dataArray);
+	histogram.classList.add('histogram');
+	document.body.appendChild(histogram);
+
+	dataArray.forEach((item) => {
+		const bar = document.createElement('div');
```

Comment:
Как ты думаешь, стоит ли выносить в отдельную функцию создание колонки?

Reply:
Думаю нет, не вижу в этом смысла

Reply:
Мне кажется, что будет неплохо, если вынести это вынести в отдельную функцию. Так как код будет более читабельный, будет "отдельный кусок" кода, который имеет свою область отвествтенности, что в будущем может помочь в дебаге. 
Да, в текущем случае, та же читаемость не так сильно увеличится, но в больших проетах это будет очень заметно. 
В целом, разбиение отдельных кусков кода на небольшие функции, является хорошей практикой

Reply:
Понял

Reply:
я тут вынес пару моментов в отдельные функции, может теперь лучше их вынести в отдельный модуль и назвать его утилиты?
```suggestion
		function validateInput(input) {
	return input
		.split(' ')
		.filter(item => !isNaN(Number(item)) && item !== '')
}

function createBarElement() {
	const bar = document.createElement('div');
	bar.classList.add('bar');
	return bar;
}
```

Reply:
Можно и отдельно вынести. 
Сразу скажу на счет имени функции, ты не инпут валидируешь, а данные

Reply:
По поводу функции, которая создает новый элемент - давай сделаем эту функцию более универсальной, чтобы они могла создавать любой элемент и вешать любой/любые классы

Reply:
Изменил название на `validateData`.
Функция теперь выглядит так:
`function createElementWithClass(element, classesArray) {
	const newElement = document.createElement(element);
	newElement.classList.add(...classesArray);
	return newElement;
}`